Chaos - FW Renegade and Heretics.cat

```diff
@@ -8638,7 +8641,7 @@
                       <rules/>
                       <infoLinks/>
                       <modifiers>
-                        <modifier type="set" field="points" value="15">
+                        <modifier type="set" field="points" value="12">
```

Comment:
This modifier isn't needed any more as the base cost for the autocannon has been updated.